pom.xml

```diff
@@ -31,10 +31,12 @@
             <artifactId>spring-boot-starter</artifactId>
```

Comment:
spring-boot-starter is a part of spring-boot-starter-web, could be omitted

Reply:
fixed